module/classes/DevModePerformance.mjs

```diff
@@ -17,14 +17,18 @@ export class DevModePerformance {
     });
   }
 
-  static actorCRUDTest = async (iterations = 1000) => {
-    console.log('running test with iterations:', `${iterations}`, !!iterations, iterations || 1000);
+  static actorCRUDTest = async (type, iterations = 1000, synthetic) => {
```

Comment:
Since this is getting more complex, lets destructure an object as the first parameter here. That will help this 'self-document' as well.

```suggestion
  static actorCRUDTest = async ({ type, iterations = 1000, synthetic } = {}) => {
```